tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -738,30 +742,30 @@
   % Find out, whether with args or not.
   \pgfkeysifdefined{\pgfkeyscurrentpath/.@args}%
   {% Yes, so add to body and reuse args
-  \pgfkeysaddvalue{\pgfkeyscurrentpath/.@body}{#1}{#2}%
-  % Redefine code
-  {%
-    \pgfkeysgetvalue{\pgfkeyscurrentpath/.@args}{\pgfkeys@tempargs}%
-    \pgfkeysgetvalue{\pgfkeyscurrentpath/.@body}{\pgfkeys@tempbody}%
-    \def\pgfkeys@marshal{\expandafter\gdef\expandafter\pgfkeys@global@temp\pgfkeys@tempargs}%
-    \expandafter\pgfkeys@marshal\expandafter{\pgfkeys@tempbody}%
-  }%
+    \pgfkeysaddvalue{\pgfkeyscurrentpath/.@body}{#1}{#2}%
+    % Redefine code
+    {%
+      \pgfkeysgetvalue{\pgfkeyscurrentpath/.@args}{\pgfkeys@tempargs}%
+      \pgfkeysgetvalue{\pgfkeyscurrentpath/.@body}{\pgfkeys@tempbody}%
+      \def\pgfkeys@marshal{\expandafter\gdef\expandafter\pgfkeys@global@temp\pgfkeys@tempargs}%
+      \expandafter\pgfkeys@marshal\expandafter{\pgfkeys@tempbody}%
+    }%
     \pgfkeysifdefined{\pgfkeyscurrentpath/.@@body}{%
-    % support for \pgfkeysndefargs:
-    \pgfkeyslet{\pgfkeyscurrentpath/.@@body}{\pgfkeys@global@temp}%
-  }{%
-    % support for \pgfkeysdefargs:
+      % support for \pgfkeysndefargs:
+      \pgfkeyslet{\pgfkeyscurrentpath/.@@body}{\pgfkeys@global@temp}%
+    }{%
+      % support for \pgfkeysdefargs:
       \pgfkeyslet{\pgfkeyscurrentpath/.@cmd}{\pgfkeys@global@temp}%
-  }%
+    }%
   }%
```

Comment:
For example, before that commit, it's hard to identify which `}` ends the second argument of `\pgfkeysifdefined`.